### DISCUSSION TITLE
Flows - documentation

### DISCUSSION THREAD

**Discussion by kropsi** [directus/directus#13804] [Archive]
I just want to say that the idea of Flows is amazing! Thank you for that functionality!

Is there any where to see after documentation. It is not obvious how to reference between the data fields. For instance I want to update a certain field (slug) with information from the other field (name). How do I reference between these? How do I reference to the ID from the trigger (the flow should be triggered by creation of the item).

![billede](https://user-images.githubusercontent.com/946662/172837270-80adc176-61ec-48c3-a9ed-00ea2e8a4952.png)

**Comment by benhaynes** [comment]
We're still finishing up and extending the Docs for Flows... but you can see the work below.

https://docs.directus.io/configuration/flows/

@erondpowell — let's make sure to have a dedicated section on the payload variables we include, and how to use/traverse those. @rijkvanzanten or @Nitwel can help you with that.

**Reply by erondpowell** [reply]
We do have something like this:  https://docs.directus.io/configuration/flows/#the-flow-object

Will improve on it more soon and link to Azzy's api docs as well, which show the objects more extensively:
https://docs.directus.io/reference/system/flows/
https://docs.directus.io/reference/system/operations/

**Reply by benhaynes** [reply]
Looking good!

**Comment by erondpowell** [comment]
@kropsi 
So every Operation has access to the entire Flow Object. 
Any value in the Flow Objects can be accessed via dot-notation. 
For example, if you want some value generated by the Trigger: 

```json
{ 
    "$trigger": 
       {
          "nested_key": "value"
       }
}
```
You can use `$trigger.nested_key` to access `value`.

Use the [Logs](https://docs.directus.io/configuration/flows/#logs) along with the [log to console](https://docs.directus.io/configuration/flows/operations/#log-to-console) Operation inspect your Object.

Please let me know if this clarifies things, or if you need more information! 😸

**Comment by nervusdm** [comment]
Bumped here.

Flow is for me, the killer feature of directus. Never seen something like this, so good.
But there is no example / doc.
Can't wait to have it.

I can' write it for you guys, since I just start..

**Reply by erondpowell** [reply]
You can find docs here:
https://github.com/directus/directus/discussions/13804#discussioncomment-2915985
